src/elements/content-sidebar/activity-feed/task-new/__tests__/Task.test.js

```diff
@@ -109,7 +109,7 @@ describe('elements/content-sidebar/ActivityFeed/task-new/Task', () => {
                 status="NOT_STARTED"
             />,
         );
-        expect(incompleteWrapper.find('.bcs-is-taskOverdue')).toHaveLength(1);
+        expect(incompleteWrapper.find('.bcs-TaskDueDate')).toHaveLength(1);
```

Comment:
I don't think this class should be here. We should use a stable reference (add a testid), or some other check.